include/antworld/render_target.h

```diff
@@ -19,9 +19,9 @@ class RenderTarget
     //------------------------------------------------------------------------
     // Public API
     //------------------------------------------------------------------------
-    void bind();
-    void unbind();
-    void clear();
+    void bind() const;
+    static void unbind();
```

Comment:
While these can technically be static (because OpenGL is a giant nightmareish, stateful blob), semantically it makes no sense i.e. calling ``RenderTarget::clear`` will clear whatever the active render target it

Reply:
I think I agree with you on this one. We don't really gain anything by doing this (cf. automatically ``const``ifying methods)